src/components/Header/Header.jsx

```diff
@@ -14,6 +14,10 @@ import AccountWidget from '../AccountWidget';
 import styles from './Header.css';
 import SearchForm from '../Search/SearchForm';
 import AppLogo from '../App/AppLogo';
+import BurgerMenu from '../BurgerMenu';
+
+import { HEADER_MD_BREAK } from '../Player/Header/PlayerHeader';
+import { HEADER_SM_BREAK } from '../Player/Header/PlayerHeader';
```

Comment:
Just keep in mind @recurza , those breakpoints were specific to how the header layout broke. I don't know if it's necessary to follow the exact same rules for the navbar. If they are the same, then I would suggest moving these out of the PlayerHeader and into some utility file or something. Also, you can do absolute paths:
```js
import { myThing } from 'components/Player/Header/PlayerHeader';
```